src/pywink/api.py

```diff
@@ -642,9 +643,11 @@ def get_devices_from_response_dict(response_dict, device_type):
     devices = []
 
     api_interface = WinkApiInterface()
+    check_list = bool(isinstance(device_type, (list,)))
```

Comment:
is this `bool(` necessary?  Last time I checked, `isinstance` already returns a bool.

Reply:
Good point I'll change that.